README.md

```diff
@@ -29,25 +29,33 @@ curl https://netbox.example.org/api/ipam/ip-addresses/?dns_name=example-vm-host
 }
 ```
 
-## Usage
+## Enabling
 
-To activate the plugin you need to compile CoreDNS with the plugin added
+To activate the *netbox* plugin you need to compile CoreDNS with the plugin added
 to `plugin.cfg`
 
 ```
 netbox:github.com/oz123/coredns-netbox-plugin
 ```
 
-Then add it to Corefile:
+### Ordering in plugin.cfg
+
+The ordering of plugins in the `plugin.cfg` file is important to ensure you
+get the behaviour you expect when using multiple plugins in a config section.
```

Comment:
So fallthrough isn't configured with `Corefile` at runtime config.
This is pretty confusing. Maybe, this should be added explicitly?
WDYT?

Reply:
So to be honest my understanding of how ordering of plugins works and fall through has improved a lot while working on this plugin…so I can’t say I know everything, but my understanding is that it works as follows:

- the ordering of plugins in `plugin.cfg` is set at compile time which hardcodes the order that requests can be handled from one plugin to the next
- actual `fallthrough` from one plugin to the next is then configured to happen at run time in the `Corefile`

This does depend on the plugin, for example the cache plugin always does `fallthrough` while `file` never does.

So in this case I have coded things up that if `fallthrough` is set, the next plugin is called to handle the request on errors or no record was found. 

I’m honestly not sure if this is the best way, but for my uses it works well.

Reply:
OK, I think I have a suggestion that can improve the documentation a bit. Instead of:
```
The behaviour you expect when using multiple plugins in a config section.
```
Try:
```
The behaviour you expect when using multiple plugins in a [Corefile server block][1].

[1]: https://coredns.io/manual/toc/#server-blocks
```

This uses the upstream terminology and makes it clearer (since, I at lease read `plugin.cfg` as `plugin-config` in my head).

Reply:
That does make things clearer I think so I've added that change.